pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "NerpyBot"
-version = "0.4.6"
+version = "0.4.8"
```

Comment:
Meinst du das kann automatisch funktionieren beim publish?

so mache ich das bei meinen Foundry mods:

`name: Module CI/CD

on:
  release:
      types: [published]

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2

    # Substitute the Manifest and Download URLs in the module.json
    - name: Substitute Manifest and Download Links For Versioned Ones
      id: sub_manifest_link_version
      uses: microsoft/variable-substitution@v1
      with:
        files: 'module.json'
      env:
        version: ${{github.event.release.tag_name}}`

https://github.com/Rikshy/pf2e-dullibrew/blob/main/.github/workflows/main.yml